server/datastore/mysql/migrations/tables/20231106144110_AddWindowsProfilesTables.go

```diff
@@ -0,0 +1,86 @@
+package tables
+
+import (
+	"database/sql"
+	"fmt"
+)
+
+func init() {
+	MigrationClient.AddMigration(Up_20231106144110, Down_20231106144110)
+}
+
+func Up_20231106144110(tx *sql.Tx) error {
+	_, err := tx.Exec(`
+ALTER TABLE mdm_apple_delivery_status RENAME TO mdm_delivery_status;
+`)
+	if err != nil {
+		return fmt.Errorf("failed to rename mdm_apple_delivery_table table: %w", err)
+	}
+	_, err = tx.Exec(`
+ALTER TABLE mdm_apple_operation_types RENAME TO mdm_operation_types;
+`)
+	if err != nil {
+		return fmt.Errorf("failed to rename mdm_apple_operation_types table: %w", err)
+	}
+
+	_, err = tx.Exec(`
+-- track the team/no-team profiles - those represent the desired state of
+-- profiles per teams.
+CREATE TABLE mdm_windows_configuration_profiles (
+  -- this is typically called just id but this is consistent with the apple
+  -- profiles table.
+  profile_id   INT(10) UNSIGNED NOT NULL AUTO_INCREMENT,
+
+  -- this is 0 for no-team, or > 0 for a given team id, same as the apple
+  -- profiles table.
+  team_id      INT(10) UNSIGNED NOT NULL DEFAULT '0',
+
+  name         VARCHAR(255) NOT NULL,
+  syncml       MEDIUMBLOB NOT NULL,
```

Comment:
I went with a `MEDIUMBLOB` instead of `BLOB` given how verbose syncml can be, I was afraid that 64KB might not be enough in edge cases.